runtime/src/bank.rs

```diff
@@ -3166,7 +3166,7 @@ impl Bank {
     }
 
     pub fn get_fee_for_message(&self, message: &SanitizedMessage) -> Option<u64> {
-        let lamports_per_signature = {
+        let _lamports_per_signature = {
```

Comment:
This function previously would return `None` if the `recent_blockhash` was not recent enough, or `nonce` is invalid/doesn't exist.
We no longer **need** to check that since it doesn't actually affect the fee.

Left as is for now. If we changed this to always return u64, it could affect RPC calls.